### PR TITLE
[dev-v2.7] Reintroduce milestone 2.7.12 changes

### PR DESCRIPTION
**Pull request by alexandreLamarre**
Requires https://github.com/rancher/charts/pull/3603

## Issue: 
Original issues

https://github.com/rancher/rancher/issues/44164
SURE-7505 

## Problem

Some patches were reverted during the fast forward of release-v2.7 to dev-v2.7 due to versioning issues on our end 

https://github.com/rancher/charts/pull/3358
https://github.com/rancher/charts/pull/3388

## Testing

See original issues

### CODE REVIEW

**Review by joshmeranda — APPROVED** [review]

**Review by adamkpickering — APPROVED** [review]

File: release.yaml [hunk 1]
```diff
@@ -1,20 +1,25 @@
...
+- 4.4.0-rc1
 rancher-cis-benchmark-crd:
-  - 4.4.0-rc1
+- 4.4.0-rc1
 rancher-istio:
-  - 102.4.0+up1.18.2
+- 102.4.0+up1.18.2
+rancher-logging:
+- 102.0.3-rc1+up3.17.10
+- 102.0.3-rc1
```
Comment (nicholasSUSE) [inline release.yaml]: Maybe I am missing something but this seems wrong

File: release.yaml [hunk 2]
```diff
@@ -1,20 +1,25 @@
...
+rancher-logging-crd:
+- 102.0.3-rc1+up3.17.10
 longhorn:
   - 102.3.2+up1.5.4
 longhorn-crd:
   - 102.3.2+up1.5.4
 rancher-monitoring:
-  - 102.0.4-rc1
+- 102.0.4-rc1+up40.1.2
+- 102.0.4-rc1
```
Comment (nicholasSUSE) [inline release.yaml]: Same as above

File: release.yaml [hunk 3]
```diff
@@ -1,20 +1,25 @@
...
 rancher-monitoring-crd:
-  - 102.0.4-rc1
-rancher-logging:
-  - 102.0.3-rc1
+- 102.0.4-rc1+up40.1.2
+- 102.0.4-rc1
```
Comment (nicholasSUSE) [inline release.yaml]: Same as above

**Review by nicholasSUSE — APPROVED** [review]
Ok I saw that you removed the weird versions in the final commit, approving.

**Comment by lucasmlp** [comment]
I would comment the same as [here](https://github.com/rancher/charts/pull/3603#issuecomment-2000525763)

**Comment by alexandreLamarre** [comment]
Superceded by 
https://github.com/rancher/charts/pull/3729
https://github.com/rancher/charts/pull/3730

**Pull request closed without merging**